oap-server/server-configuration/configuration-etcd/src/main/java/org/apache/skywalking/oap/server/configuration/etcd/EtcdServerSettings.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.configuration.etcd;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.skywalking.oap.server.library.module.ModuleConfig;
+
+/**
+ * @author Alan Lau
+ */
+@ToString
+@Getter
+@Setter
+public class EtcdServerSettings extends ModuleConfig {
+
+    private String clusterName = "default";
+    private String serverAddr;
+    private String group;
+    private int period = 5;
```

Comment:
default is 5, unit? If unit=second, I think it is too short for timeout, try 30 for configuration center, because in most case, people don't treat performance and latency very seriously for it.

Reply:
For the config
```yaml
#    period : 60 # Unit seconds, sync period. Default fetch every 60 seconds.
```

Why don't use 60 in codes?

Reply:
ok, i will fix this.

Reply:
@wayilau I saw that you changed the default value of `EtcdServerSettings#period` to 30, but in the `application.yml`, it's still:

```yml
#  etcd:
#    period : 60 # Unit seconds, sync period. Default fetch every 60 seconds.
```

it's quite misleading, please keep consistent

Reply:
ok